projects/observability/src/shared/components/table/data-cell/exit-calls/exit-calls-table-cell-renderer.component.test.ts

```diff
@@ -37,7 +41,11 @@ describe('Exit Calls table cell renderer component', () => {
       providers: [tableCellRowDataProvider({ apiExitCalls: 2, apiCalleeNameCount: value })]
     });
 
-    expect(spectator.component.getMaxShowAPICalleeNameCount(value)).toMatchObject(value);
-    expect(spectator.component.totalCountOfDifferentAPICallee).toBe(2);
+    expect(spectator.component.apiCalleeNameCount).toMatchObject([
```

Comment:
This test has value, but what would be even better would be to verify that this data renders - for example by querying for the rendered spans and checking their content (e.g. `expect(spectator.queryAll('.api-callee-name')[0]).toContainText('key1');`)

That checks both how the data is processed but also that the template is correctly binding what we expect.

Reply:
Okay, this change I can make.

Reply:
Nevermind, discussed offline - because this is a template that's not rendered in the scope of this component, it's ugly to test and not worth it.